source/pattern-library/cards/utilization-trend-card/index.md

```diff
@@ -93,3 +95,4 @@ layout: page
     [ usability content goes here ]
   </div>
 </div>
+<script src="/ngdocs/formatForPFsite.js"></script>
```

Comment:
Ended up just adding a script line to the header file, like we're doing for the other scripts.